src/components/CorporateCampaign/LoanSearch/LoanSearchFilters.vue

```diff
@@ -648,6 +651,14 @@ export default {
 	}
 }
 
+.divider {
```

Comment:
It looks like this class is no longer used.

Reply:
Agree, this could be removed in some later cleanup